### PR TITLE
[shared] Bump OTLP proto files to 1.5.0

### PR DESCRIPTION
**Pull request by Kielek**
Handles https://github.com/open-telemetry/opentelemetry-proto/releases/tag/v1.5.0

## Changes

Bump OTLP proto files to 1.5.0

## Merge requirement checklist

* [x] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (license requirements, nullable enabled, static analysis, etc.)
* ~~[ ] Unit tests added/updated~~
* ~~[ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes~~
* ~~[ ] Changes in public API reviewed (if applicable)~~

### CODE REVIEW

File: src/Shared/Proto/opentelemetry/proto/logs/v1/logs.proto [hunk 1]
```diff
@@ -208,4 +210,18 @@ message LogRecord {
...
+  //
+  // Recommended to be fully qualified and short (no longer than 256 characters).
+  //
+  // Presence of event_name on the log record identifies this record
+  // as an event.
+  //
+  // [Optional].
+  //
+  // Status: [Development]
+  string event_name = 12;
```
Comment (Kielek) [inline src/Shared/Proto/opentelemetry/proto/logs/v1/logs.proto:226]: @rajkumar-rangaraj, should be add support for this in the OTLP exporter?

Comment (cijothomas) [reply src/Shared/Proto/opentelemetry/proto/logs/v1/logs.proto:226]: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol#environment-variables-1
It'll be bit awkward that EventName gets exported automatically, but EventId still requires opt-in. But still, I think that is the way forward.

**Review by cijothomas — APPROVED** [review]
Adding EventName can be a separate PR if we chose to add it.

**Comment by Kielek** [comment]
#6040 - added to Future mailstone

File: src/Shared/Proto/opentelemetry/proto/logs/v1/logs.proto (hunk 1 above)
Comment (rajkumar-rangaraj) [reply src/Shared/Proto/opentelemetry/proto/logs/v1/logs.proto:226]: @Kielek Not before the next stable release. This was not a part of the earlier code. I see you had created an issue to track this item; we could add it after this stable release.

**Pull request merged**